src/main/scala/com/decodified/scalassh/Command.scala

```diff
@@ -18,11 +18,26 @@ package com.decodified.scalassh
 
 import java.io.{ByteArrayInputStream, File, FileInputStream, InputStream}
 import net.schmizz.sshj.connection.channel.direct.Session
-
-final case class Command(command: String, input: CommandInput = CommandInput.NoInput, timeout: Option[Int] = None)
+trait Command {
+  val command: String
+  val input: CommandInput  = CommandInput.NoInput
+  val timeout: Option[Int] = None
+  val safe: Boolean
+}
+case class SafeCommand(command: String,
+                       override val input: CommandInput = CommandInput.NoInput,
+                       override val timeout: Option[Int] = None,
+                       safe: Boolean = true)
+    extends Command
+case class UnsafeCommand(command: String,
```

Comment:
The two sub-types of the ADT here are structually identical, so there appears to be little value of refactoring to an ADT in the first place.
Also, what would `SafeCommand(..., safe = false)` or `UnsafeCommand(..., safe = true)` signify?

It seems we can accomplish everything you want to do by simply adding one boolean flag to the existing `Command` case class, no?

Reply:
This is all gone...